contrib/pg_tde/documentation/docs/command_line_tools.md

```diff
@@ -0,0 +1,39 @@
+# Command line tools
+
+The `pg_tde` extension provides new command line tools and modifies some existing tools to work with encrypted WAL and tables.
+
+## pg_tde_change_key_provider
+
+A tool for modifying the configuration of a key provider, possibly also changing its type.
+
+This tool edits the configuration files directly, ignoring permissions or running postgres processes.
+It's only intended use is to fix servers that can't be started up because of inaccessible key providers:
+for example, when restoring an old backup, when the address of the key provider changed ib the meantime, the tool can be used to correct the configuration, allowing the server to start up.
+
+It should not be used when the server is up and running, in that case, `pg_tde` provides the `pg_tde_change_key_provider_<type>` SQL functions instead.
+
+### Usage
+
+```
+pg_tde_change_key_provider [-D <datadir>] <dbOid> <provider_name> <new_provider_type> <provider_parameters...>
```

Comment:
I think we also need to explain `dbOid`

Reply:
Agreed, I also thought the same thing while reading through it.